lib/svix.ts

```diff
@@ -31,3 +31,10 @@ export const listWebhooks = async (appId: string) => {
 export const deleteWebhook = async (appId: string, endpointId: string) => {
   return await svix.endpoint.delete(appId, endpointId);
 };
+
+export const createEvent = async (name: string, description: string) => {
+  return await svix.eventType.create({
```

Comment:
Here you are creating an eventType, instead you should use the message endpoint to send the message to the Webhook endpoint. 

Here is the API doc https://api.svix.com/docs#tag/Message/operation/create_message_api_v1_app__app_id__msg__post

We're already creating an `appId` for each team using the `teamId`.

Reply:
So only 3 things are required to send an event.

- app Id
- event type
- payload